### PR TITLE
style(icons): updates icons

### PR DESCRIPTION
**Pull request by antlio**
this pr updates library icons in order to fix viewbox issue spotted by @cameronrohani  + restyle some of them.

**⊢ collection icon before / after**
<div>
image
image
</div>

as i removed some we are not filling up the list anymore, might want to replace them too?

### CODE REVIEW

**Review by cameronrohani — APPROVED** [review]

**Review by amritk — COMMENTED** [review]
will defer to @cameronrohani and @hwkr

**Comment by cameronrohani** [comment]
almost perfect, just two icons that aren't working with the stroke so we should probably change out

image
image

**Review by cameronrohani — CHANGES_REQUESTED** [review]
packages/icons/src/icons/image-picture-flower.svg
packages/icons/src/icons/tag-new-circle.svg

need some tweaks

File: packages/icons/src/icons/programming-computer-database-server.svg
Comment (hwkr) [inline packages/icons/src/icons/programming-computer-database-server.svg:1]: I feel like still having a server icon for the user selectable ones would be nice 🤔

File: packages/icons/src/icons/shopping-cart.svg
Comment (hwkr) [inline packages/icons/src/icons/shopping-cart.svg:1]: Maybe this should be renamed `shopping-bag` if we're changing how it looks?

**Review by hwkr — CHANGES_REQUESTED** [review]
The icons look good! Thanks for cleaning those up.

I think if we're deleting icons we have the potential to break people's collections though, we probably either need to add a fallback for the library icon or a migration that updates peoples icons.

**Comment by antlio** [comment]
> packages/icons/src/icons/image-picture-flower.svg packages/icons/src/icons/tag-new-circle.svg
> 
> need some tweaks

fixed!

File: packages/icons/src/icons/shopping-cart.svg
Comment (antlio) [reply packages/icons/src/icons/shopping-cart.svg:1]: revert to cart!

File: packages/icons/src/icons/programming-computer-database-server.svg
Comment (antlio) [reply packages/icons/src/icons/programming-computer-database-server.svg:1]: added back

**Review by hwkr — APPROVED** [review]
Look good! 😊

**Review by hanspagel — APPROVED** [review]
Nice update!

**Pull request merged**